src/allencell_ml_segmenter/core/file_input_widget.py

```diff
@@ -261,10 +266,11 @@ def _process_checked_signal(self, row: int, state: Qt.CheckState) -> None:
                 )
 
     def _reset_channel_combobox(self) -> None:
-        self._channel_select_dropdown.clear()
-        self._channel_select_dropdown.setPlaceholderText("")
-        self._channel_select_dropdown.setCurrentIndex(-1)
-        self._channel_select_dropdown.setEnabled(False)
+        if self._include_channel_selection:
```

Comment:
All the detail about channel selection also seem out of place here.

Reply:
^^ that idea sounds good to me, if it's not too tricky to implement widget inheritance